data/seriesRepository.go

```diff
@@ -628,7 +628,7 @@ func (r *SeriesRepository) GetSeriesObservations(
 	var percent sql.NullBool
 	YOY, YTD, C5MA := YOYPercentChange, YTDPercentChange, Centered5MA
 
-	err = r.DB.QueryRow(`SELECT measurements.percent
+	err = r.DB.QueryRow(`SELECT series.percent
 	FROM series
 	LEFT JOIN measurement_series ON measurement_series.series_id = series.id
```

Comment:
Now we don't need the measurement joins in this query. So we can delete line 633 and 634, and the query should still work. The other joins look fine.